Gearbox linear pool (#2)
joaobrunoah/balancer-v2-monorepo: pkg/deployments/tasks/20221116-gearbox-rebalanced-linear-pool/readme.md

# 2022-11-16 - Gearbox Rebalanced Linear Pool

> Deployment of the `GearboxLinearPoolFactory`, for Linear Pools with a wrapped aToken. This new deployment includes a permissionless rebalancing contract that has Asset Manager privileges.

Reviewer: ```suggestion
Deployment of the `GearboxLinearPoolFactory`, for Linear Pools with a Gearbox Diesel Token (dToken).
```

## Useful Files
- [`GearboxLinearPool` ABI](./abi/GearboxLinearPool.json)
- [`GearboxLinearPoolFactory` ABI](./abi/GearboxLinearPoolFactory.json)

> - [`GearboxLinearPoolRebalancer` ABI](./abi/GearboxLinearPoolRebalancer.json)

Reviewer: None of these files are included in this PR, but I think they should be.
